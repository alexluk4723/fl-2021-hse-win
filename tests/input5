ui               := ^dkk^
d := (^ui^ | ui) @ gdfkfkfk sfgkdfklgmdfkg
@*
	fggkdljgkdfjg
		fgfdjgkdfgj
		dfgkgjdfl;gk
	fgk
	@ -- LOVUSHKA!
	zdfksggkf
*@
@ And and and
@*
	ok
*@
return ui
