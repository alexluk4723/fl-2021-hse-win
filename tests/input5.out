ui               := ^dkk^
d := (^ui^ | ui) 












return ui
